test_fms/interpolator/test_interpolator2.F90

```diff
@@ -90,8 +90,8 @@ program test_interpolator2
   NAMELIST / test_interpolator_nml / test_file_daily_noleap, test_file_daily_julian, &
                                      test_file_yearly_noleap, test_file_yearly_julian, test_file_no_time
 
-  if(lkind==r4_kind) tol=1.e-4_r8_kind
-  if(lkind==r8_kind) tol=1.e-6_r8_kind
+  if(lkind==r4_kind) tol=1.e-1_r8_kind
+  if(lkind==r8_kind) tol=1.e-1_r8_kind
```

Comment:
These tolerance values are quite different than the ones being replaced.  Is the 0.1 tolerance good enough?  If it is, I don't think there is a need for the two type checks on lkind and you can get by with a simple assignment.

Reply:
This PR reverts back to the original way time is converted when the file and model calendars are different.  In the original, the conversion only uses the number of days to convert the time, not the number of days and seconds and unless the unit tests are wrong, it leads to some conversion inaccuracy I guess.....